testing/windows_local_audio_client.py

```diff
@@ -0,0 +1,52 @@
+import os
+import sys
+import struct
+import numpy as np
+import time
+import requests
+import pyaudiowpatch as pyaudio
+
+def translator():
+    DOCKER_IP = "http://0.0.0.0:8000/"
+    print("TRANSLATOR IS RUNNING")
```

Comment:
Bad practice to use print statements like this. Use the [`logging`](https://docs.python.org/3/library/logging.html) module instead